.github/workflows/dotnet.yml

```diff
@@ -25,15 +27,17 @@ jobs:
     - name: Build (TestFlashLFQ)
       run: cd mzLib && dotnet build --no-restore ./TestFlashLFQ/TestFlashLFQ.csproj
     - name: Add coverlet collector (Test)
-      run: cd mzLib && dotnet add Test/Test.csproj package coverlet.collector
+      run: cd mzLib/Test && dotnet add package coverlet.collector
     - name: Add coverlet collector (TestFlashLFQ)
-      run: cd mzLib && dotnet add TestFlashLFQ/TestFlashLFQ.csproj package coverlet.collector
+      run: cd mzLib/TestFlashLFQ && dotnet add package coverlet.collector
     - name: Test
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./Test/Test.csproj
+      run: cd mzLib/Test && dotnet test --no-build --verbosity normal 
+        --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
     - name: TestFlashLFQ
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./TestFlashLFQ/TestFlashLFQ.csproj
-    - name: Codecov
+      run: cd mzLib/TestFlashLFQ && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
```

Comment:
The TestFlashLFQ coverage dropped a bunch, so I'm not sure this is working https://app.codecov.io/gh/smith-chem-wisc/mzLib/pull/669